### PR TITLE
[doc] add primus-rooms-redis-adapter

### PR DESCRIPTION
**Pull request by fadeenk**
Add primus-rooms-redis-adapter to the list of community plugins

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1966,6 +1966,20 @@ README.md and see it be merged automatically.
 </dl>
 
 <dl>
+  <dt><a href="https://github.com/fadeenk/primus-rooms-redis-adapter">primus-rooms-redis-adapter</a></dt>
+  <dd>
+    A redis adapter for promis-rooms module. Supports integration with metroplex and omega-supreme.
```
Comment (lpinca) [inline README.md]: s/promis-rooms/primus-rooms/

Comment (fadeenk) [inline README.md]: Sorry but I am not quite understanding what you want me to do here

Comment (fadeenk) [inline README.md]: oops been staring at it for a while and did it see it. Just updated it :)

**Comment by lpinca** [comment]
Awesome, thanks!

**Pull request merged**